Controller/GameController.java

```diff
@@ -25,8 +26,28 @@ public GameController(int viewType, int boardType) {
     }
     public void initialize() {
         Timer timer = new Timer();
-        long oneSecond = 1000;
+        long oneSecond = 10000;
         RefreshMapTask task = new RefreshMapTask(viewInterface, board);
         timer.schedule(task, 0, oneSecond);
     }
+    public void handleUserInput() {
+        viewInterface.displayHelp();
+        boolean consoleIsOpen = true;
+        Scanner userInput = new Scanner(System.in);
+        Command command = new Command();
+        while (consoleIsOpen) {
+            String[] splitCommands = Console.parse(userInput.nextLine());
+            char action = command.getAction(splitCommands);
+            // If action is not found in command list
+            if (action == '#') {
+                viewInterface.displayInvalidCommand();
+                continue;
+            }
+            String argsCheck = command.checkArgs(splitCommands, action);
```

Comment:
Can we change this to create a command object that has attributes for location information, argument validity and any error message? This looks like it can be encapsulated in an object for better readabilty/usage